lib/nanoc/extra/link_collector.rb

```diff
@@ -4,6 +4,18 @@
 
 module ::Nanoc::Extra
   class LinkCollector
+
+    @@uri_attrs = {
```

Comment:
I’d prefer to have this in a constant rather than a class variable (I think Rubocop will complain anyway).